make/Makefile.dev

```diff
@@ -0,0 +1,60 @@
+DOCKER_REPO?=quay.io/openshiftio
+IMAGE_NAME?=git-service
+SHORT_COMMIT=$(shell git rev-parse --short HEAD)
+ifneq ($(GITUNTRACKEDCHANGES),)
+SHORT_COMMIT := $(SHORT_COMMIT)-dirty
+endif
+
+TIMESTAMP:=$(shell date +%s)
+TAG?=$(SHORT_COMMIT)-$(TIMESTAMP)
+
+DEPLOY_DIR:=deploy
+
+.PHONY: create-resources
+create-resources:
+	@echo "Logging using system:admin..."
+	@oc login -u system:admin
+	@echo "Creating sub resources..."
+	@echo "Creating CRDs..."
+	@oc create -f https://raw.githubusercontent.com/redhat-developer/devopsconsole-operator/master/deploy/crds/devopsconsole_v1alpha1_gitsource_crd.yaml
+	@echo "Creating Namespace"
+	@oc create -f $(DEPLOY_DIR)/namespace.yaml
+
+.PHONY: create-cr
+create-cr:
+	@echo "Creating Custom Resource..."
```

Comment:
what else is it supposed to do?

Reply:
It was a placeholder but we probably won't need it anyway. Removed it in https://github.com/redhat-developer/git-service/pull/2/commits/c31c4860ecf89798868488dc28c8def6c9fb4d95